packages/sit-onyx/src/components/TestInput/TestInput.vue

```diff
@@ -31,15 +63,54 @@ const handleChange = (event: Event) => {
   const target = event.target as HTMLInputElement;
   emit("change", target.value);
 };
+
+onMounted(() => {
+  /* element will always be truthy onMounted */
+  const element = coreElement.value;
+  if (element) {
+    /* we need to watch for value changes, 
+    using computed on element.validity.valid 
+    unfortunately misses the validity resetting */
+    watch(
+      value,
+      () => {
+        /* todo we could use a composable or util for the "only update when it changed" pattern? */
```

Comment:
remove the check, instead create a second watcher for emitting the event only when the valid state changed.